Yoast/YoastSEO.js
# Use keyword density instead of topic density when synonyms are set

## Summary

This PR can be summarized in the following changelog entry:

* Removes the topic density assessment and uses the keyword density assessment also when synonyms are set.

## Test instructions

This PR can be tested by following these steps:

* Make a beta version using `release/7.8` for Yoast SEO Premium and this branch for YoastSEO.js
* Add a text with > 100 words. Set a focus keyword and a synonym.
* Add the focus keyword and the synonym in the text.
* Make sure that the keyword density assessment is triggered also after you've added the synonym. It should only show keyword occurrences (both in terms of counting as well as in terms of 

Fixes #1577
